src/styles/index.scss

```diff
@@ -1,3 +1,44 @@
-body {
+@import './header';
+@import './cards';
+@import './stars';
+
+* {
+  box-sizing: border-box;
+  display: block;
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  max-width: 1024px;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
+  display: grid;
+
+  --column-count: 1;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    &__container {
+      --column-count: 2;
+    }
```

Comment:
The use of `&__container` is unnecessary here. Since you are already within the `.container` block, you can directly set `--column-count: 2;` without using `&__container`.